src/eip7594/eip7594.c

```diff
@@ -612,6 +669,47 @@ static C_KZG_RET compute_commitment_to_aggregated_interpolation_poly(
     return ret;
 }
 
+/**
+ * Compute weighted sum of proofs
```

Comment:
```suggestion
 * Compute weighted sum of proofs.
```

Reply:
Addressed in https://github.com/ethereum/c-kzg-4844/commit/e68de38097d2db24e2b3b243e1635b84c814b2c4